tap_hubspot/__init__.py

```diff
@@ -94,7 +98,7 @@ class StateFields:
     "contact_lists":        "/contacts/v1/lists",
     "forms":                "/forms/v2/forms",
     "workflows":            "/automation/v3/workflows",
-    "owners":               "/owners/v2/owners",
+    "owners":               f"/crm/v3/owners?limit={OWNERS_LIMIT}&archived=false"
```

Comment:
Why the query parameters `limit` and `archived`? Shouldn't we fetch everything possible?

Reply:
Maximum limit is 500, and it's by default 100 (so it needs to be specified)
I just put archived for specificity. Archived=false gives us the most results

Reply:
So why don't we put 500? (And I would put a `TODO` saying that we might need to handle pagination for this endpoint in the future if we pass the 500 threshold)

I don't understand how archived=false gives us more results than not specifying archived at all? I thought it would act as a filter